client-app/src/common/models/search/paged-search-query.ts

```diff
@@ -1,12 +1,12 @@
-import { Dictionary } from '../dictionary';
-import { ISearchCriteria } from './search-criteria';
+import { Dictionary } from '@common/models/dictionary';
+import { IPagedSearchCriteria } from '@common/models/search/paged-search-criteria';
 
 /* eslint @typescript-eslint/camelcase: ["error", {properties: "never"}] */
-export class SearchQuery {
+export class PagedSearchQuery {
   page?: string;
   page_size?: string;
 
-  toSearchCriteria<TSearchCriteria extends ISearchCriteria>(searchCriteriaType: { new(): TSearchCriteria }): TSearchCriteria {
+  toSearchCriteria<TSearchCriteria extends IPagedSearchCriteria>(searchCriteriaType: { new(): TSearchCriteria }): TSearchCriteria {
     const searchCriteria = new searchCriteriaType();
     searchCriteria.pageNumber = this.page ? Number(this.page) : undefined;
     searchCriteria.pageSize = this.page_size ? Number(this.page_size) : undefined;
```

Comment:
can be replaced with separate function:
searchCriteria.pageNumber = convertToNumber(this.page);	    
searchCriteria.pageSize = convertToNumber(this.page_size);

Reply:
toDictionary() can be moved to utils

Reply:
Fixed